applications/ParticleMechanicsApplication/python_scripts/particle_mpm_solver.py

```diff
@@ -364,10 +364,10 @@ def _execute_check_and_prepare(self):
     def _import_constitutive_laws(self):
         materials_filename = self.settings["material_import_settings"]["materials_filename"].GetString()
         if (materials_filename != ""):
-            import read_materials_process
             # Add constitutive laws and material properties from json file to model parts.
-            read_materials_process.ReadMaterialsProcess(self.model, self.settings["material_import_settings"])
-
+            material_settings = KratosMultiphysics.Parameters("""{"Parameters": {"materials_filename": ""}} """)
+            material_settings["Parameters"]["materials_filename"].SetString(materials_filename)
+            KratosMultiphysics.ReadMaterialsUtility(material_settings, self.model)
```

Comment:
@loumalouomega why did you not pass the `self.settings["material_import_settings"]` in StructuralMechanics?
@bodhinandach just copied this but I am not sure what was your initial intention

Reply:
That was also my question. But apparently the default parameter inside the function is designed to take:
```
"Parameters": {
        "materials_filename": "materials.json"
    }
```
not
`"materials_filename": "materials.json"`
So I followed the one in StructuralMechanics

Reply:
hmm strange
@loumalouomega @marandra why did you change the input-syntax in the c-utility?

Reply:
I will just merge this at the moment. Please kindly notify me if you think to modify the syntax in the future.